Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: get update fmt lint test
+
+GO       := GO111MODULE=on GOPRIVATE=github.com/linkedin GOSUMDB=off go
+GOBUILD  := CGO_ENABLED=0 $(GO) build $(BUILD_FLAG)
+GOTEST   := $(GO) test -gcflags='-l' -p 3 -v -race
+
+FILES    := $(shell find core -name '*.go' -type f -not -name '*.pb.go' -not -name '*_generated.go' -not -name '*_test.go')
+TESTS    := $(shell find core -name '*.go' -type f -not -name '*.pb.go' -not -name '*_generated.go' -name '*_test.go')
+
+get:
+	$(GO) get ./...
+	$(GO) mod verify
+	$(GO) mod tidy
+
+update:
+	$(GO) get -u -v all
+	$(GO) mod verify
+	$(GO) mod tidy
```

Comment:
I ran locally `make update` and got this new updates, not sure if you'd like to include them
![image](https://user-images.githubusercontent.com/950087/72373481-5b3f5580-36d6-11ea-84e8-e310025a26a8.png)

Reply:
Will include, thanks 👍 Those are kind of libs that pretty much updated on a daily basis, so you're almost guaranteed to have them somewhat out of date.

Reply:
ok, so not need to update them unless you see some security fix on the new releases